packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -118,11 +116,22 @@ function getCurrentAction(prevActName) {
   return actions[prevActName]?.action
 }
 
-function getHoveredAtomId(atoms: Map<number, ReAtom>): number | null {
-  for (const [id, atom] of atoms.entries()) {
-    if (atom.hover) return id
+function getHoveredItemsIds(
+  ctab: Record<string, Map<number, Record<string, unknown>>>
+): Record<string, number[]> {
+  const hoveredItems = {}
```

Comment:
Didn't quite get it.
The whole idea of this function is to transform the data into another format.
As to number/array thing, I already changed it so it would give back single item. That's more expected, agreed.

Reply:
In the previous version we returned either a number or zero (false). Now we always return a true value (number or object), it seems to me it can lead to errors that will be difficult to detect

Reply:
Moved boolean check inside a function. Now it complies with your suggestion.